engine/src/main/java/org/terasology/rendering/dag/nodes/BlurPassesNode.java

```diff
@@ -61,10 +66,9 @@
     public void initialise() {
         renderingConfig = config.getRendering();
         blur = worldRenderer.getMaterial("engine:prog.blur");
-        requireFBO(new FBOConfig("sceneBlur0", 0.5f, FBO.Type.DEFAULT));
-        requireFBO(new FBOConfig("sceneBlur1", 0.5f, FBO.Type.DEFAULT));
-        requireFBO(new FBOConfig("sceneOpaque", 1.0f, FBO.Type.HDR).useDepthBuffer().useNormalBuffer().useLightBuffer().useStencilBuffer());
-        requireFBO(new FBOConfig("sceneToneMapped", 1.0f, FBO.Type.HDR));
+        requireDynamicFBO(new FBOConfig(BLUR_0_URN, 0.5f, FBO.Type.DEFAULT));
+        requireDynamicFBO(new FBOConfig(BLUR_1_URN, 0.5f, FBO.Type.DEFAULT));
+        requireDynamicFBO(new FBOConfig(TONE_MAPPED_URN, 1.0f, FBO.Type.HDR));
```

Comment:
I'm not convinced by all these _URN suffix. An urn is basically a string. And any IDE can tell you its type if really necessary. I'd remove them all.